docs/dev_guide/newcomers.rst

```diff
@@ -110,6 +111,14 @@ If you see that astropy_helpers is different or missing please run ::
 
 which should fix the issue for you.
 
+Every once in a while you may have to update this submodule. This can be
+done with ::
+
+    git submodule update
+
+This will solve what git will consider a difference in the astropy_helper
+signature.
+
```

Comment:
This is a repeat of the block of text above. Maybe merge them into a clearer paragraph?